config/riff-build.yaml

```diff
@@ -473,6 +473,14 @@ metadata:
     component: build.projectriff.io
   name: riff-build-manager-role
 rules:
+- apiGroups:
```

Comment:
Can you also run and commit the results from `make package`, this change should also end up in `config/riff-build.yaml`

Reply:
done